LongWarOfTheChosen/Src/WotCWeaponsAndArmor/Classes/X2DownloadableContentInfo_WotCWeaponsAndArmor.uc

```diff
@@ -0,0 +1,38 @@
+//---------------------------------------------------------------------------------------
+//  FILE:   XComDownloadableContentInfo_WaroftheChosenRebalance.uc                                    
+//           
+//	Use the X2DownloadableContentInfo class to specify unique mod behavior when the 
+//  player creates a new campaign or loads a saved game.
+//  
+//---------------------------------------------------------------------------------------
+//  Copyright (c) 2016 Firaxis Games, Inc. All rights reserved.
+//---------------------------------------------------------------------------------------
+
+class X2DownloadableContentInfo_WotCWeaponsAndArmor extends X2DownloadableContentInfo;

```

Comment:
Is this class needed? I don't think it is if it doesn't do anything. Not sure though.

Reply:
I don't think I need this class right now but I am going to leave it for the time being as I might need to use it for some last minute processing later.